torch/csrc/jit/codegen/cuda/test/test_gpu_gather_ops.cpp

```diff
@@ -0,0 +1,278 @@
+#if defined(USE_CUDA)
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
```

Comment:
Such a large set of `#include`, are we for sure that all the included files are necessary?

Reply:
It's not checked, it's just imitating the other test file.